x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/_classification_exploration.scss

```diff
@@ -1,4 +1,19 @@
-.euiFormRow.mlDataFrameAnalyticsClassification__actualLabel {
-  padding-top: $euiSize * 4;
+/* Workaround for EuiDataGrid within a Flex Layout */
+.mlDataFrameAnalyticsClassification {
+  width: calc(100% - 0px);
```

Comment:
why do you need `calc` here? also, keep in mind it's not supported by IE11 properly https://caniuse.com/#feat=calc

Reply:
The `calc` is used to work around an issue with `EuiDataGrid` and percentage based width of the wrapping container when used in flex layouts. The original issue in the EUI repo is here: https://github.com/elastic/eui/issues/2618

`calc` in IE has limitations but this one should work hopefully. I cannot test it at the moment because it seems Kibana is broken for IE11 on `master`. Will try again once I hear back from QA team.

Reply:
Managed to test in IE. Pushed some fixes in de5e1c0 and updated the PR description with before and after screenshots.

Reply:
After the refactor to not use flex anymore for the layout, the `calc()` is no longer necessary. Removed in 7905eec.